site/tutorials/keptn-litmus-018.md

```diff
@@ -0,0 +1,378 @@
+
+    ```
+    kubectl create namespace litmus-chaos
+    ```
+
+1. We also need to create the custom resources for the experiments we want to run later, as well as some permissions.
+
+    ```
+    kubectl apply -f ./litmus/pod-delete-ChaosExperiment-CR.yaml 
+
+    kubectl apply -f ./litmus/pod-delete-rbac.yaml 
+    ```
+
+## Setup Prometheus
+Duration: 3:00
+
+Before we are going to create the project with Keptn, we'll install the Prometheus integration to be ready to fetch the data that is later on needed for the SLO-based quality gate evaluation. 
+
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite. 
+
+* To install the Prometheus and Alert Manager, execute:
+<!-- command -->
+```
+kubectl create ns monitoring
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm install prometheus prometheus-community/prometheus --namespace monitoring
+```
+
+### Execute the following steps to install prometheus-service
+
+* Download the Keptn's Prometheus service manifest
+
+```
+helm upgrade --install -n keptn prometheus-service \
+https://github.com/keptn-contrib/prometheus-service/releases/download/0.9.1/prometheus-service-0.9.1.tgz \
+--reuse-values \
+--set prometheus.namespace="monitoring" \
+--set prometheus.endpoint="http://prometheus-server.monitoring.svc.cluster.local:80" \
+--set prometheus.namespace_am="monitoring"
+```
+
+### Optional: Verify Prometheus setup in your cluster
+
+* To verify that the Prometheus scrape jobs are correctly set up, you can access Prometheus by enabling port-forwarding for the prometheus-service:
+<!-- command -->
+```
+kubectl port-forward svc/prometheus-server 8080:80 -n monitoring
+```
+
+
+## Setup Litmus integration
+Duration: 1:00
+
+Similar to the Prometheus integration, we are now adding the Litmus integration. This integration will be responsible to trigger the experiments with Litmus and listens for `sh.keptn.event.test.triggered` events that are sent from Keptn.
+
+This can be done via the following command.
+
+```
+kubectl apply -f ../deploy/service.yaml
+```
+
+We now have all the integrations installed and connected to the Keptn control plane. Let's move on with setup up a project!
+
+## Create project
+Duration: 1:00
+
+A project in Keptn is the logical unit that can hold multiple (micro)services. Therefore, it is the starting point for each Keptn installation.
+We have already cloned the demo resources from Github, so we can go ahead and create the project.
+
+
+**Recommended:** Create a new project with Git upstream:
+
+To configure a Git upstream for this tutorial, the Git user (`--git-user`), an access token (`--git-token`), and the remote URL (`--git-remote-url`) are required. If a requirement is not met, go to [the Keptn documentation](https://keptn.sh/docs/0.9.0/manage/git_upstream/) where instructions for GitHub, GitLab, and Bitbucket are provided.
```

Comment:
```suggestion
To configure a Git upstream for this tutorial, the Git user (`--git-user`), an access token (`--git-token`), and the remote URL (`--git-remote-url`) are required. If a requirement is not met, go to [the Keptn documentation](https://keptn.sh/docs/0.19.0/manage/git_upstream/) where instructions for GitHub, GitLab, and Bitbucket are provided.
```
You had 0.9.0 so I made it Release 19 but since you are doing this exercise with Release 18, perhaps you want to point to this page in the 0.18.x tree.  The same page with the same information is there.